web/controllers/unit_controller.ex

```diff
@@ -10,7 +10,8 @@ defmodule OrgtoolDb.UnitController do
   end
 
   def index(conn, _params, _current_user, _claums) do
-    units = Repo.all(Unit) |> Repo.preload([:unit_type])
+    units = Repo.all(Unit)
+    |> Repo.preload([:unit_type, :unit, :units, :members, :leaders, :applicants])
```

Comment:
this is not here on purpose, this is A LOT of data when we got many units which are not visible. Those fields are displayed when a speicific unit is read however. This allows to load just the units that are currently displayed by just reading those.